consensus/ibft/src/test/java/org/hyperledger/besu/consensus/ibft/validation/SignedDataValidatorTest.java

```diff
@@ -47,9 +48,12 @@
   private final KeyPair proposerKey = KeyPair.generate();
```

Comment:
Since the pattern new BouncyCastleNodeKey(KeyPair.generate()) is used so often, perhaps a static generate method on BouncyCastleNodeKey would clean test code a bit